# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../blueprobe/install.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- blueprobe-0.19/Makefile.vanilla	2008-09-01 09:35:15.000000000 +0200
+++ blueprobe-0.19/Makefile	2008-09-01 09:35:21.000000000 +0200
@@ -32,7 +32,7 @@
 	install -d $(DESTDIR)/etc/init.d
 	install $(PACKAGE).init $(DESTDIR)/etc/init.d/$(PACKAGE)
 	install -d $(DESTDIR)/etc/rcS.d
-	ln -s ../init.d/$(PACKAGE) $(DESTDIR)/etc/rcS.d/S97blueprobe
+	ln -sf ../init.d/$(PACKAGE) $(DESTDIR)/etc/rcS.d/S97blueprobe
 
 clean:
 	rm -f $(PACKAGE)
